backend/src/models/role.model.js

```
@@ -0,0 +1,24 @@
+    }
+  }
+
+  static async getAllRoles() {
+    try {
+      const roles = await prisma.user_roles.findMany();
+      return roles;
+    } catch (error) {
+      throw new Error(`Error fetching roles: ${error.message}`);
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)